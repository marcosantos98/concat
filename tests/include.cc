"./std.cc" include

"Sizeof(i8) == " print i8 sout 10 putc
"Sizeof(i16) == " print i16 sout 10 putc
"Sizeof(i32) == " print i32 sout 10 putc
"Sizeof(i64) == " print i64 sout 10 putc
"Sizeof(int) == " print int sout 10 putc
