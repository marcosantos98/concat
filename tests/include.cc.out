Sizeof(i8) == 1
Sizeof(i16) == 2
Sizeof(i32) == 4
Sizeof(i64) == 8
Sizeof(int) == 8
